### PR TITLE
CMD/Account - Coverage test issue

### LINKED ISSUE #984: potentially broken tests in cmd/gossamer account
## Expected Behavior

Test of command gossamer account should confirm that it works properly under different situations(options, flags, etc.)

## Current Behavior

I checked in my device(MacBook Pro) what part of account.go file is covered and saw that some core parts are not covered, for example, there is a test scenario which supposedly tests command gossamer account --generate, however, this is what I get from coverage:
image

I think in the tests the cli.Context is not created correctly because after creation the arguments of the context (ctx.Args()) is empty.

## Possible Solution

One quick possible solution is to actually simulate by using the app:
```go
// TestAccountGenerate test "gossamer account --generate"
func TestAccountGenerate(t *testing.T) {
	testDir := utils.NewTestDir(t)
	defer utils.RemoveTestDir(t)

	directory := fmt.Sprintf("--basepath=%s", testDir)

	err := app.Run([]string{"irrelevant", "account", directory, "--generate=true", "--password=false"})
	if err != nil {
		t.Fatal(err)
	}
}
```
This indeed tests the command because that part of code is covered:
image

## Steps to Reproduce

If this is not happening only in my device, then you can reproduce it by:
1. Go to the cmd/gossamer directory
2. Run in the terminal > go test -coverprofile=coverage.out
3. Run in the terminal > go tool cover -html=coverage.out
4. When the page is opened in your browser, check the account.go file

## Specification

- go version: go1.14.4 darwin/amd64
- gossamer --version: 0.0.1
- operating system: macOS

## Checklist

- [x] I have read [CODE_OF_CONDUCT](https://github.com/ChainSafe/gossamer/blob/development/.github/CODE_OF_CONDUCT.md) and [CONTRIBUTING](https://github.com/ChainSafe/gossamer/blob/development/.github/CONTRIBUTING.md) 
- [ ] I have provided as much information as possible and necessary
- [ ] I am planning to submit a pull request to fix this issue myself

### PR DESCRIPTION
**Pull request by RyRy79261**
## Changes

- Implemented solution to extend coverage 

## Tests

```
   cd cmd/gossamer 
   go test -coverprofile=coverage.out
   go tool cover -html=coverage.out
   //  When the page is opened in your browser, check the account.go file

```

## Checklist

- [x] I have read [CODE_OF_CONDUCT](https://github.com/ChainSafe/gossamer/blob/development/.github/CODE_OF_CONDUCT.md) and [CONTRIBUTING](https://github.com/ChainSafe/gossamer/blob/development/.github/CONTRIBUTING.md) 
- [x] I have provided as much information as possible and necessary
- [x] I have reviewed my own pull request before requesting a review
- [ ] All integration tests and required coverage checks are passing

## Issues

closes #984

### CODE REVIEW

**Review by noot — COMMENTED** [review]
could you also do the same fix for the other tests in `cmd/gossamer/account_test.go`?

**Comment by RyRy79261** [comment]
@noot I've added the solution with the relevant flags however I'm unsure what exactly the solution is doing or how to assess it's success

File: cmd/gossamer/account_test.go [hunk 1]
```diff
@@ -26,6 +27,8 @@ import (
 func TestAccountGenerate(t *testing.T) {
 	testDir := utils.NewTestDir(t)
 	defer utils.RemoveTestDir(t)
+	directory := fmt.Sprintf("--basepath=%s", testDir)
+	err := app.Run([]string{"irrelevant", "account", directory, "--generate=true", "--password=false"})
```
Comment (noot) [inline cmd/gossamer/account_test.go:31]: please check error, same for other tests

**Review by noot — APPROVED** [review]

**Review by edwardmack — APPROVED** [review]

File: cmd/gossamer/account_test.go [hunk 2]
```diff
@@ -95,7 +108,11 @@ func TestAccountGenerateType(t *testing.T) {
 func TestAccountImport(t *testing.T) {
 	testDir := utils.NewTestDir(t)
 	defer utils.RemoveTestDir(t)
-
+	directory := fmt.Sprintf("--basepath=%s", testDir)
+	err := app.Run([]string{"irrelevant", "account", directory, "--import"})
```
Comment (noot) [inline cmd/gossamer/account_test.go:112]: you need to provide a value for `--import`, previously it was `testfile`

**Comment by RyRy79261** [comment]
Expanding the tests for this & hunting down some test keystores for the import tests

File: cmd/gossamer/account_test.go (hunk 2 above)
Comment (RyRy79261) [reply cmd/gossamer/account_test.go:167]: @noot Where is the testfile sourced?

**Review by arijitAD — APPROVED** [review]

**Pull request merged**